pegasus/sites.v3/code.org/announcements.json

```diff
@@ -0,0 +1,24 @@
+{
+  "pages": {
+    "/home": "teacher-apps-open-2020",
+    "/courses": "teacher-apps-open-2020"
+  },
+  "banners": {
+    "teacher-apps-open-2020": {
+      "image": "https://code.org/images/professional-learning-2019-3.png",
```

Comment:
These image is very large.  We should probably use use Pegasus' `fit-` or `fill-` feature (in the URL itself) to reduce the dimensions of the image that is retrieved.  It can also be used to crop the image while maintaining the correct aspect ratio without modifying the source image itself, which is handy in case we want to use the same image in different situations.

cc @agealy